workers/unity/Assets/Playground/Scripts/MonoBehaviours/WorldCommandHelper.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+
+namespace Playground.MonoBehaviours
+{
+    public class WorldCommandHelper : IDisposable
```

Comment:
I see what this class is trying to do and why it's useful but I can't help but wonder whether something is wrong with our API in the first place if we need something like this. What was the original idea behind separating command sender and command response receiver requirables? Can we not have something for matching command contexts under the hood so users don't need to manually create and dispose them themselves? What was the idea behind not having an API that roughly looks like this again?
`void SendSomeCommand(argsRelatedToTheCommand, someCallbackToBeCalledAsSoonAsTheCommandReturns)`.